test/system/700-play.bats

```diff
@@ -411,6 +411,23 @@ _EOF
     run_podman rmi -f userimage:latest
 }
 
+@test "podman kube play --replace external storage" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
+    run_podman play kube $PODMAN_TMPDIR/test.yaml
+    # Force removal of container
+    run_podman rm --force -t0 test_pod-test
+    # Create container external using buildah with same name
```

Comment:
However, I do find `container external` confusing. Should that be `external container`?